### PR TITLE
[Synapse]-Add scenario test for workspace, sqlpool, sparkpool and firewall.

### PR DESCRIPTION
**Pull request by sunsw1994**
Add scenario test for synapse management sdk.
 - workspace
 -  sqlpool
 - sparkpool
 - firewall.

### CODE REVIEW

File: sdk/synapse/Microsoft.Azure.Management.Synapse/tests/ScenarioTests/WorkspaceOperationTests.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                if (workspaceGet.ProvisioningState.Equals("Succeeded"))
+                {
+                    Assert.Equal(CommonTestFixture.WorkspaceType, workspaceGet.Type);
+                    Assert.Equal(workspaceName, workspaceGet.Name);
+                    Assert.Equal(CommonData.Location, workspaceGet.Location);
+                    break;
+                }
+
+                Thread.Sleep(30000);
+                Assert.True(i < 60, "Synapse Workspace is not in succeeded state even after 30 min.");
```
Comment (idear1203) [inline sdk/synapse/Microsoft.Azure.Management.Synapse/tests/ScenarioTests/WorkspaceOperationTests.cs:37]: >min [](start = 95, length = 3)

seconds?

Comment (sunsw1994) [reply sdk/synapse/Microsoft.Azure.Management.Synapse/tests/ScenarioTests/WorkspaceOperationTests.cs:37]: The information will be shown when the condition is false. In that case, i = 60, that means `60*30000ms=>60*30s=>30mins`.

Comment (idear1203) [reply sdk/synapse/Microsoft.Azure.Management.Synapse/tests/ScenarioTests/WorkspaceOperationTests.cs:37]: Alright. Thanks for the explanation..

**Review by idear1203 — APPROVED** [review]

**Review by isra-fel — APPROVED** [review]
LGTM. THanks

**Pull request merged**